tests/test_payments.py

```diff
@@ -63,6 +64,12 @@ def test_cancel_payment(client, response):
     assert canceled_payment.id == PAYMENT_ID
 
 
+def test_cancel_paymanet_invalid_id(client):
```

Comment:
Spelling error: `paymanet` => `payment`

Reply:
I fixed this spelling mistake, but probably in another PR.